whipper/command/cd.py

```diff
@@ -476,18 +484,35 @@ def _ripIfNotRipped(number):
                     tries -= 1
                     logger.critical('giving up on track %d after %d times',
                                     number, tries)
-                    raise RuntimeError("track can't be ripped. "
-                                       "Rip attempts number is equal to %d",
-                                       self.options.max_retries)
-                if trackResult.testcrc == trackResult.copycrc:
-                    logger.info('CRCs match for track %d', number)
+                    if self.options.keep_going:
+                        logger.warning("track %d failed to rip. "
+                                       "Continuing to next track" % number)
```

Comment:
Shouldn't next track be `number + 1`?
Is this warning going to be issued even for the last track? (referencing a next track which does not exists)

Reply:
I can change this to remove the reference to the next track.